APE21.rst

```diff
@@ -0,0 +1,187 @@
+  non-trivial.
+* Even once fixes are backported to the LTS branch, carrying out releases is not
+  fully automated and requires manual steps, so having to do
+  bugfix releases for both the latest release branch and LTS branch puts an extra
+  burden on the core package release team.
+
+In addition, it is not clear that LTS releases are needed anymore - now that the
+core package is over ten years old, it has reached a much higher level of
+stability. Most packages in the scientific Python ecosystem do not have LTS
+releases, and there is no longer a strong argument for having this
+in astropy either.
+
+Proposed release procedure
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This APE proposes the following:
+
+* No longer designate any releases as LTS.
+* Keep the 6-month release cycle and bump releases to x.0 every 2 years (with
+  the possibility of allowing this to be more or less than 2 years if there is a
+  consensus amongst core package maintainers that this can be changed)
+* Refer to the x.0 releases every 2 years (e.g., v6.0) as "major"
+  releases, other releases every 6 months (e.g., v6.1) as "minor", and bug fixing
+  releases (e.g., v6.1.2) as "bugfix" releases.
+* Minimize as much as possible any changes that break API or change results in
+  minor releases, but be pragmatic and occasionally allow such changes if they would
+  not affect many users.
+* Add deprecation warnings or future warnings in minor and major releases but only
+  go through with the breaking changes in major releases (either the next major
+  release following when a warning was added, or subsequent ones if the changes
+  are deemed to be high impact enough to wait longer).
+
+With this in place, we would effectively be striving to maximize stability over
+two-year periods but users would see deprecation warnings to prepare them for
+changes. This is in contrast to the current LTS system where users might not see
+a deprecation warning until the next LTS release as deprecations are not typically
+backported.
+
+Guidelines on deprecations and API changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The main proposed guidelines are as follows:
+
+* API changes should only be carried out in major releases.
+* Any API changes should be preceded by deprecation warnings that should be
+  visible to users in at least one minor release cycle prior to the API change
+  being carried out.
+* New deprecations should not be introduced in bugfix releases - this will
+  ensure that deprecations will therefore be emitted for at least 6 months
+  before a major release.
+* If developers wish to make an API change at a point in time where the next
+  release is a major release, they should introduce the deprecation in the major
+  release and carry out the change in the following major release.
+* The following will not be considered to be part of the API and can therefore
+  be changed in bugfix and minor releases:
```

Comment:
Maybe slightly tighter: "... changed in minor releases (and spelling corrections, etc., also in bugfix releases)."

Reply:
Am fine with removing any suggestion to make spelling changes in bugfix releases - my real purpose is just to minimize the burden of making trivial changes. And I agree we should really make sure that bugfix releases do not break any code (to the extent it is under our control of course).